index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Microverse Partnership: Ademola-Agasha</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div>
+            <header>
+                <h1 id="title" class="text-center">Ademola-Agasha Microverse Project
+                </h1>
+                <p id="description" class="text-center">we are software developers
+                </p>
+            </header>
+        </div>
+        <div>
+            <form id="survey-form" class="form">
+                <div class="form-group">
```

Comment:
In this section, you can use semantic tags in order to better explain the relationship between different sections of the webpage. You can use the section tag for instance in this part, instead of a div. You can read more about semantic tags [here](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md).